pokecli.py

```diff
@@ -176,6 +176,13 @@ def init_config():
         type=float,
         default=15.0
     )
+    parser.add_argument(
+        "-hr",
+        "--health_record",
+        help="Send anonymous bot event to GA for bot heath record, furthur ML will depend on it. Set \"health_record\":false if you need disable it.",
+        type=bool,
+        default=True
```

Comment:
This should be defaulted to false.  We should encourage people to opt-in, but it should never be the default for tracking.